src/foam/nanos/crunch/ui/CapabilityInterceptView.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.crunch.ui',
+  name: 'CapabilityInterceptView',
+  extends: 'foam.u2.Element',
+
+  implements: [ 'foam.mlang.Expressions' ],
+
+  requires: [
+    'foam.nanos.crunch.Capability'
+  ],
+
+  imports: [
+    'stack'
+  ],
+
+  properties: [
+    { name: 'data' },
```

Comment:
If just providing a name, you can just use a string directly.

```suggestion
    'data' ,
```